### DISCUSSION TITLE
enable cutting only if `cciprev > management.cuttings.ccut/100`  ?

### DISCUSSION THREAD

**Discussion by gabo-di** [KUL-RSDA/AquaCrop#345] [Q&A] [answered]
Hello all, when we perform a cutting, the canopy cover only changes if `cciprev > management.cuttings.ccut/100`, but we can record a harvest (and change the biomass) even if this is not the case. Does that make sense?    around here  are the lines in question  run.f90:7231 https://github.com/KUL-RSDA/AquaCrop/blob/6b242b6650bce6f1387668aa30f40f0c293a4b8d/src/run.f90#L7231

**Comment by mbechtold** [comment, marked as answer]
I forward here a response from Dirk Raes:
The code is correct. The reasoning is as follows:
At each harvest, the canopy cover becomes management.cuttings.ccut/100, (e.g. CCcut = 30 %). The total biomass produced since the last cut is harvested on that day (and recorded as the harvest). If due to stresses, the Canopy Cover does not increase after a cut (hence CC remains CCcut = 30 %), that does not imply that the crop does not produce biomass (however, it will be small since CC and Transpiration are small). At a next cut (planned at a specific day), the (little) biomass, produced between the 2 cuts, will be reported as harvest.
It is not the case that biomass only increases if CC also increases, because in AquaCrop the production of biomass is linked to crop transpiration.
A crop with a constant CC still transpires and hence produce biomass.
Note that even when the crop reaches its maximum canopy cover (CCx) early in the season (e.g. 85 %), the next days still a lot of biomass will be produced.
In AquaCrop you can plan cuts at specific days (Multiple harvest are specified in the Field management) = timing is specified by the user. This looks like what you are simulating and you will end up with what is explained above. However it is more reasonable to consider that farmers will only harvest if sufficient biomass is produced. Therefore in Field Management the option is available to generate harvests by a criterion (e.g. 4 ton/ha dry biomass produced between cuts within a specific time window). If specified as such, it is very likely that when due to stresses the Canopy Cover does not increase and remains at CCcut, the farmer will not have to go back to the field for a next cut as long as the biomass production remains below the specified threshold (e.g. 4 ton/ha).
Consult section 2.13.5 Multiple harvests of Chapter 2 of the AquaCrop Reference manual for more details.
